R/radsl.R

```diff
@@ -76,9 +76,9 @@ radsl <- function(N = 400, # nolint
     set.seed(seed)
   }
 
-  # stop if race_list  lengths do not match race probability length
+  # stop if race_list length do not match race probability length
```

Comment:
Just needs a minor change of the verb and the documentation then needs re-rendering :)
```suggestion
  # stop if race_list length does not match race probability length
```